### ISSUE TITLE
Editor forgets custom map data source

### ISSUE THREAD

**Issue by nc011** [openstreetmap/iD#9864]
### How to reproduce the issue?

Open iD
Add URL to URL box for custom map data source
End editing session and leave iD editor (can close tab, or just click on "openstreetmap" logo)
Go back to iD editor and click on custom map data source
Custom URL box is empty.
 

### Screenshot(s) or anything else?

Previously (assume previous release), a custom map data URL would remain in the URL box. This allowed users to open up the same custom map data without re-pasting in the URL.

However, now (presumably due to this latest release) the custom URL is not saved in the URL box between editing sessions. 

Assuming this isn't an issue my end, this seems like a regression in functionality. 

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit

### What version numbers does this issue effect?

2.27.1

### Which browsers are you seeing this problem on?

Firefox

**Comment by nc011** [comment]
Just to note that custom background URL _is_ remembered across editing sessions.

**Comment by tordans** [comment]
Hey! Given the last comment I will close this because I am quite sure the feature does work. Please re-open in case there is still an issue.

FYI: If you use the prerelease version of iD from https://github.com/openstreetmap/iD/?tab=readme-ov-file#participate, you will be able to bookmark a custom imagery URL because that is part of the URL then.

There is https://github.com/openstreetmap/openstreetmap-website/pull/3033 which tried to improve the UX on the website to also have this feature…

**Comment by Dimitar5555** [comment]
@tordans The issue is about custom **map data URL**. The inconsistency is that the custom background URL is remembered while the custom map data URL isn't.

**Comment by tyrasd** [comment]
Yeah, it would make sense for the custom map _data_ to behave similarly like the custom map _background_ functionality. I.e. that it remembers the last used endpoint. Slightly related: it is also irritating that the custom map data input field is cleared every time the modal window is opened.


//edit: it might also make sense to allow loading of gpx, geojson, etc. via an URL.
